tests/parametric/test_sampling_span_tags.py

```diff
@@ -0,0 +1,141 @@
+    if dm is not None or "meta" in first_span:
+        assert first_span["meta"].get(SAMPLING_DECISION_MAKER_KEY) == dm
+    assert parent_span["metrics"].get(SAMPLING_PRIORITY_KEY) == parent_priority
+    assert parent_span["metrics"].get(SAMPLING_RULE_PRIORITY_RATE) == parent_rate
+    assert child_span.get("metrics", {}).get(SAMPLING_RULE_PRIORITY_RATE) is None
+    if child_span != first_span:
+        assert child_span.get("meta", {}).get(SAMPLING_DECISION_MAKER_KEY) is None
+
+
+@scenarios.parametric
+class Test_Sampling_Span_Tags:
+    @pytest.mark.parametrize("library_env", [{"DD_TRACE_SAMPLE_RATE": 1}])
+    @bug(library="python", reason="Python sets dm tag on child span")
+    @bug(library="python_http", reason="Python sets dm tag on child span")
+    @bug(library="nodejs", reason="NodeJS does not set priority on parent span")
+    @bug(library="ruby", reason="ruby does not set dm tag on first span")
+    @bug(library="dotnet", reason="dotnet does not set dm tag on first span")
+    def test_tags_child_dropped_sst001(self, test_agent, test_library):
+        with test_library:
+            with test_library.start_span(name="parent", service="webserver") as parent_span:
+                with test_library.start_span(
+                    name="child", service="webserver", parent_id=parent_span.span_id
+                ) as child_span:
+                    child_span.set_meta(MANUAL_DROP_KEY, None)
+
+        traces = test_agent.wait_for_num_spans(2, clear=True)
+
+        parent_span = find_span_in_traces(traces, Span(name="parent", service="webserver"))
+        child_span = find_span_in_traces(traces, Span(name="child", service="webserver"))
+
+        _assert_sampling_tags(parent_span, child_span, traces[0][0], "-3", 2, 1)
+
+    @pytest.mark.parametrize("library_env", [{"DD_TRACE_SAMPLE_RATE": 1}])
+    @bug(library="python", reason="Python sets dm tag on child span")
+    @bug(library="python_http", reason="Python sets dm tag on child span")
+    @bug(library="ruby", reason="ruby does not set dm tag on first span")
+    @bug(library="dotnet", reason="dotnet does not set dm tag on first span")
+    @bug(library="nodejs", reason="nodejs sets dm tag -4 on first span")
+    def test_tags_child_kept_sst007(self, test_agent, test_library):
+        with test_library:
+            with test_library.start_span(name="parent", service="webserver") as parent_span:
+                with test_library.start_span(
+                    name="child", service="webserver", parent_id=parent_span.span_id
+                ) as child_span:
+                    child_span.set_meta(MANUAL_KEEP_KEY, None)
+
+        traces = test_agent.wait_for_num_spans(2, clear=True)
+
+        parent_span = find_span_in_traces(traces, Span(name="parent", service="webserver"))
+        child_span = find_span_in_traces(traces, Span(name="child", service="webserver"))
+
+        _assert_sampling_tags(parent_span, child_span, traces[0][0], "-3", 2, 1)
```

Comment:
Can anyone speak to whether this assertion is correct, specifically with respect to the `_sampling_priority: 2` expectation here and in `sst001`? It's not clear to me why `_sampling_priority` should be the same when the child span is `MANUAL_DROP`ed as when it's `MANUAL_KEEP`'ed.

@dgoffredo @cbeauchesne